mobilevit/models/mobilevit.py

```diff
@@ -29,9 +31,7 @@ def get_training_model(
     input_layer = keras.Input(shape=image_shape)  # Input Layer
 
     # Convolutional Stem Stage
-    x = conv_block(
-        input_layer=input_layer, num_filters=configs.out_channels[0], name="conv_stem_"
-    )
+    x = conv_block(input_layer=input_layer, num_filters=configs.out_channels[0])
 
     for i in range(4):
```

Comment:
Let's assign `4` to a variable with meaningful name?

Reply:
Can you suggest something?